src/server.js

```diff
@@ -70,29 +70,70 @@ app.get('/accounts', (req, res) => {
     let chifra = spawn("chifra", ['accounts', req.query.search1, req.query.search2, debug, '--nocolor'], {env: env});
     chifra.stderr.pipe(process.stderr);
     chifra.stdout.pipe(res).on('finish', (code) => {
-        console.log(`"chifra names" exiting: ${code}`);
-        console.log(`child process exited with code ${code}`);
-        return res.end();
+        reportAndSend("accounts", code, res);
     })
 })
 
 app.get('/blocks/:id', (req, res) => {
-    let chifra = spawn("chifra", ['blocks', '--list', '--data', debug, '--nocolor'], { env: env });
+    var id = "";
+    if (typeof req.params.id != undefined)
+        id = req.params.id;
+    let chifra = spawn("chifra", ['data', '--blocks', `${id}`, debug, '--nocolor'], { env: env });
     chifra.stderr.pipe(process.stderr);
     chifra.stdout.pipe(res).on('finish', (code) => {
-        console.log(`"chifra blocks" exiting: ${code}`);
-        console.log(`child process exited with code ${code}`);
-        return res.end();
+        reportAndSend("blocks", code, res);
+    })
+})
+
+app.get('/trans/:id', (req, res) => {
```

Comment:
`/tx/:id` perhaps? Not a blocking consideration.

Reply:
I will create an issue to track this. It's a bit larger than just this one route.